src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import './App.css';
+import Navbar from './containers/navbar/Navbar';
+import BookPage from './pages/books/BookPage';
+import AuthorsPage from './pages/authors/AuthorsPage';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<BookPage />} />
+      <Route path="/author" element={<AuthorsPage />} />
```

Comment:
- [ ] Like the previous reviewer mentioned, this route should be named ```Categories``` and should lead to the category page according to the requirements.